src/app/dashboard-widgets/work-item-widget/work-item-widget.component.html

```diff
@@ -12,7 +12,7 @@ <h3>Create Work Items</h3>
       </p>
       <div class="f8-blank-slate-main-action">
         <button id="spacehome-workitems-create-button" class="btn btn-primary btn-lg"
-                [routerLink]="[{ outlets: { action: 'add-work-item' } }]">Create work item</button>
+                [routerLink]="['add-work-item']">Create work item</button>
```

Comment:
When choosing to create a work item from the new work-item-widget, I encounter the same error below. The same empty page displays with "Choose a new work item type" and nothing more.

ERROR Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'.

Reply:
Both of those exist currently, the logged error isn't displayed in production however.

The first bug is noted here: https://github.com/openshiftio/openshift.io/issues/3300

And the second is noted here: https://github.com/fabric8-ui/fabric8-planner/issues/2571